src/main/webapp/view/transcript/transcript.js

```diff
@@ -171,3 +173,36 @@ function scrollToTopOfTranscript(transcriptLine) {
   const ulElementOffset = transcriptLine.parentElement.offsetTop;
   transcriptContainer.scrollTop = transcriptLine.offsetTop - ulElementOffset;
 }
+
+/**
+ * Returns the next transcript line based on `currentTimeMs`.
+ */
+function getNextTranscriptLine(currentTimeMs) {
+  const nextTranscript = currentTranscriptLine.nextElementSibling;
+  // Video is playing normally.
+  if (isWithinTimeRange(currentTimeMs, nextTranscript)) {
+    return nextTranscript;
+  }
+  return findClosestTranscriptLine(currentTimeMs);
+}
+
+/**
+ * Searches for and returns the closest transcript line
+ * based on `timeMs`.
+ */
+function findClosestTranscriptLine(timeMs) {
+  let transcriptLinePointer = document.getElementsByTagName('li')[0];
+  while (transcriptLinePointer != null &&
+         !isWithinTimeRange(timeMs, transcriptLinePointer) &&
+         isBeforeTimeMs(transcriptLinePointer, timeMs)) {
```

Comment:
Could you switch the params of one of these so the order of parameters is consistent?

Reply:
Ideally all `time, transcriptline` functions should be in the same order. Helps with readability

Reply:
done, thanks!